src/main/java/liquibase/addFhirGlobalLibraryRows.v3.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet author="mat_dev_user" id="1" context="prod">
+        <sql>
+            <![CDATA[
+ALTER table RECENT_CQL_ACTIVITY_LOG MODIFY CQL_ID VARCHAR(64);
+DELETE FROM CQL_LIBRARY WHERE ID IN ('AdultOutpatientEncounters-FHIR4-1-1-000','AdvancedIllnessandFrailtyExclusion-FHIR4-4-0-000','FHIRHelpers-4-0-0','Hospice-FHIR4-1-0-000','MATGlobalCommonFunctions-FHIR4-4-0-000','NCQA-Common-FHIR4-5-1-000','SupplementalDataElements-FHIR4-1-0-0','TJCOverall-FHIR4-4-0-000','VTEICU-FHIR4-3-1-000');
+
+INSERT INTO CQL_LIBRARY(ID, MEASURE_ID, SET_ID, CQL_NAME, DRAFT, VERSION, FINALIZED_DATE, RELEASE_VERSION, OWNER_ID, LOCKED_USER, LOCKED_OUT_DATE, CQL_XML, REVISION_NUMBER, QDM_VERSION, LAST_MODIFIED_ON, LAST_MODIFIED_BY, LIBRARY_MODEL, FHIR_VERSION) VALUES
+('AdultOutpatientEncounters-FHIR4-1-1-000', null, '225b9882-8a50-44a7-88e6-139ad91bfc5c', 'AdultOutpatientEncounters_FHIR4', 1, 1.001, null, 'v6.0', '402803826529d99f0165d450f7d02f61', null, null, '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><cqlLookUp><library>AdultOutpatientEncounters_FHIR4</library><version>1.1.000</version><usingModel>FHIR</usingModel><usingModelVersion>4.0.0</usingModelVersion><cqlContext>Patient</cqlContext><codeSystems/><valuesets/><codes/><parameters><parameter id="1df5258a-ddde-4b71-b61d-07a20b078c76" name="Measurement Period" readOnly="true"><logic>Interval&lt;DateTime&gt;</logic></parameter></parameters><definitions/><functions/><includeLibrarys/></cqlLookUp>', 0, null, now(), '402803826529d99f0165d450f7d02f61', 'FHIR', '4.0.0');
+
+INSERT INTO CQL_LIBRARY(ID, MEASURE_ID, SET_ID, CQL_NAME, DRAFT, VERSION, FINALIZED_DATE, RELEASE_VERSION, OWNER_ID, LOCKED_USER, LOCKED_OUT_DATE, CQL_XML, REVISION_NUMBER, QDM_VERSION, LAST_MODIFIED_ON, LAST_MODIFIED_BY, LIBRARY_MODEL, FHIR_VERSION) VALUES
+('AdvancedIllnessandFrailtyExclusion-FHIR4-4-0-000', null, '0445f523-6e51-4de8-9487-920e977ed928', 'AdvancedIllnessandFrailtyExclusion_FHIR4', 1, 4.000, null, 'v5.8', '402803826529d99f0165d450f7d02f61', null, null, '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><cqlLookUp><library>AdvancedIllnessandFrailtyExclusion_FHIR4</library><version>4.0.000</version><usingModel>FHIR</usingModel><usingModelVersion>4.0.0</usingModelVersion><cqlContext>Patient</cqlContext><codeSystems/><valuesets/><codes/><parameters><parameter id="1df5258a-ddde-4b71-b61d-07a20b078c76" name="Measurement Period" readOnly="true"><logic>Interval&lt;DateTime&gt;</logic></parameter></parameters><definitions/><functions/><includeLibrarys/></cqlLookUp>', 0, null, now(), '402803826529d99f0165d450f7d02f61', 'FHIR', '4.0.0');
```

Comment:
release version v5.8 is correct, right?

Reply:
Should be v6.0 just changed it great catch!